docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -1134,6 +1138,41 @@ Result:
 └──────────────────────────────────────────────────────────────┘
 ```
 
+## addDate
```

Comment:
The original issue only talked about `addDate` but for consistency it would be nice to implement `subDate` as well.